book/src/backup.md

```diff
@@ -0,0 +1,206 @@
+
+> [!IMPORTANT]  
+> Not recommended for large sites or when you need deduplication or distributed storage, as it may cause timeouts and performance issues.
+
+## 1. Configuration
+
+ **Customize** `config/backup.php` in your editor and adjust settings as needed; inline notes outline the available configuration parameters.
+
+ **Key structure**:  
+
+- **`backup`**  
+
+    - **`name`**   
+
+    - **`source`**   
+        - **`files`**  
+
+            Specifies which directories and files to `include` and which to `exclude` in the backup.
+
+            ```php
+                'include' => [
+                    base_path(),           
+                 ],
+
+                'exclude' => [
+                    base_path(),
+                    base_path('vendor'),
+                    base_path('node_modules'),
+                    base_path('storage'),
+                    base_path('public/vendor/joypixels'), 
+                 ],
+            ```       
+
+           - **`follow_links`** 
+
+           - **`ignore_unreadable_directories`** 
+
+           - **`relative_path`** 
+
+        - **`databases`**  
+
+            Specifies the database connections to back up.
+
+    - **`database_dump_compressor`**  
+
+         Compressor class (e.g. `Spatie\DbDumper\Compressors\GzipCompressor::class`) or `null` to disable.
+
+    - **`destination`**  
+
+         Defines the storage location for backup files.​
+
+    - **`temporary_directory`**  
+
+     Staging directory for temporary files.
+
+- **`notifications`**  
+
+    Define when and how backup events trigger alerts via mail, Slack, or custom channels. 
+
+- **`monitor_backups`**  
+
+    Detect backup issues; triggers `UnhealthyBackupWasFound` when needed.  
+
+- **`cleanup`**  
+
+    Define how long to keep backups and when to purge old archives. 
+
+    - **`strategy`**  
+
+    - **`default_strategy`**  
+
+         Keeps all backups for 7 days; then retains daily backups for 16 days, weekly for 8 weeks, monthly for 4 months, and yearly for 2 years. Deletes old backups exceeding 5000 MB.
+
+         ```php
+             'keep_all_backups_for_days'                => 7,
+             'keep_daily_backups_for_days'              => 16,
+             'keep_weekly_backups_for_weeks'            => 8,
+             'keep_monthly_backups_for_months'          => 4,
+             'keep_yearly_backups_for_years'            => 2,
+             'delete_oldest_backups_when_using_more_megabytes_than' => 5000,
+         ```
+
+- **`security`**  
+
+    Ensure that only someone with your `APP_KEY` can decrypt and restore snapshots.
+
+## 2. Create a Backup
+
+**Note:** You can access the built-in Backups dashboard from the Staff menu. It shows each backup’s status, health, size, and count, and lets administrators launch unscheduled full, database, or files-only backups instantly. Another approach is to use the command line.
+
+- **Run** the Artisan command:
```

Comment:
Maybe note the three different commands here.

backup:run = full database and files backup

backup:run --only-db = only database backup

backup:run --only-files = only files backup